src/hdtSkyrimSystem.cpp

```
@@ -5,38 +5,43 @@
+#if defined(__AVX2__) || defined(__AVX512F__)
+#	include <immintrin.h>
+#else
+// float32
+// Martin Kallman
+//
+// Fast half-precision to single-precision floating point conversion
+//  - Supports signed zero and denormals-as-zero (DAZ)
+//  - Does not support infinities or NaN
+//  - Few, partially pipelinable, non-branching instructions,
+//  - Core operations ~6 clock cycles on modern x86-64
+static void __float32(float* __restrict out, const uint16_t in)
```

Clang (14.0.6)

[warning] declaration uses identifier '__float32', which is a reserved identifier

(bugprone-reserved-identifier)